src/containers/CommunityPostListSection/CommunityPost/CommunityPost.tsx

```diff
@@ -97,15 +88,53 @@ const CommunityPost = forwardRef<any, CommunityPostType>(function CommunityPost(
 
   const handleLiked = async (e: React.MouseEvent<HTMLElement>) => {
```

Comment:
```suggestion
  const handleLiked = async () => {
```
event 객체 사용 안해서 제거해도 괜찮을 것 같습니다ㅣ!